auslib/test/web/test_client.py

```diff
@@ -906,11 +906,12 @@ def testCacheControlIsNotSetFor404(self):
         ret = self.client.get('/whizzybang')
         self.assertEqual(ret.headers.get("Cache-Control"), None)
 
-    def testCacheControlIsNotSetFor500(self):
+    def testCacheControlIsSetFor500(self):
         with mock.patch('auslib.web.views.client.ClientRequestView.get') as m:
```

Comment:
I don't think test has any value anymore. It's purpose was to check the response headers for Cache-Control, but we don't get a response with headers to check now. Removing it is the right thing to do.